src/Stepper/Stepper.js

```diff
@@ -23,9 +23,9 @@ class Stepper extends Component {
      */
     activeStep: PropTypes.number,
     /**
-     * Should be two or more `<Step />` components
+     * Should be two or more `<Step />` components, but also accepts one.
      */
-    children: PropTypes.arrayOf(PropTypes.node),
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]),
```

Comment:
So `node` is essentially `any-react-component` (which can have children, or be an array). Cool. Tested it and it works indeed, both views show no warning locally.